tests/queries/0_stateless/01045_array_zip.reference

```diff
@@ -1,2 +1,3 @@
 [('a','d'),('b','e'),('c','f')]
 [('a','d','g'),('b','e','h'),('c','f','i')]
+[()]
```

Comment:
It actually returns `[]` and that's why FastTest has failed, and it makes sense to me that there shouldn't be an empty tuple when there are no arrays at all